std_msgs/Header header

uint8 qualifierSteeringAngleFrontAxle
float32 steeringAngleFrontAxleErrAmp
uint8 steeringAngleFrontAxleErrAmpInvalidFlag
float32 steeringAngleFrontAxle
uint8 steeringAngleFrontAxleInvalidFlag
uint8 steeringAngleFrontAxleEventDataQualifier
